honeycombio/resource_board.go

```diff
@@ -0,0 +1,154 @@
+package honeycombio
+
+import (
+	"encoding/json"
+	"errors"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/validation"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+	honeycombio "github.com/kvrhdn/go-honeycombio"
```

Comment:
A small formatting issue, goimports expects imports to be grouped by stdlib / other packages.
```suggestion
	"encoding/json"
	"errors"

	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
	"github.com/hashicorp/terraform-plugin-sdk/helper/validation"
	honeycombio "github.com/kvrhdn/go-honeycombio"
```